### ISSUE TITLE
fontconfig issue under macOS

### ISSUE THREAD

**Issue by suxpert** [Psychtoolbox-3/Psychtoolbox-3#677]
On macOS, It seems that the `libptbdrawtext_ftgl64.dylib` hard-coded the freetype and fontconfig location:
```bash
$ otool -L libptbdrawtext_ftgl64.dylib
libptbdrawtext_ftgl64.dylib:
	libptbdrawtext_ftgl64.dylib (compatibility version 0.0.0, current version 0.0.0)
	/System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL (compatibility version 1.0.0, current version 1.0.0)
	/opt/X11/lib/libfontconfig.1.dylib (compatibility version 11.0.0, current version 11.2.0)
	/opt/X11/lib/libfreetype.6.dylib (compatibility version 19.0.0, current version 19.6.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.50.4)
```
which leading to the OFTGL plugin fail to load when XQuartz was not installed:
```
PTB-WARNING: DrawText: Failed to load external drawtext plugin [dlopen(/Applications/Psychtoolbox/PsychBasic/PsychPlugins/libptbdrawtext_ftgl64.dylib, 10): Library not loaded: /opt/X11/lib/libfontconfig.1.dylib
  Referenced from: /Applications/Psychtoolbox/PsychBasic/PsychPlugins/libptbdrawtext_ftgl64.dylib
  Reason: image not found]. Retrying under generic name [libptbdrawtext_ftgl64.dylib].
PTB-WARNING: DrawText: Failed to load external drawtext plugin 'libptbdrawtext_ftgl64.dylib' [dlopen(libptbdrawtext_ftgl64.dylib, 10): image not found]. Reverting to legacy text renderer.
PTB-WARNING: DrawText: Functionality of Screen('DrawText') and Screen('TextBounds') may be limited and text quality may be impaired.
PTB-WARNING: DrawText: Type 'help DrawTextPlugin' at the command prompt to receive instructions for troubleshooting.
```

But in `help DrawTextPlugin`, it said that
> If you have GStreamer installed (help GStreamer) for multi-media support, then
  you may not need XQuartz, as GStreamer also provides the required libraries.

I've installed the latest GStreamer (1.18.0), which ships with freetype 24 and fontconfig 12:
```
/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/libfontconfig.1.dylib (compatibility version 12.0.0, current version 12.1.0)
/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/libfreetype.6.dylib (compatibility version 24.0.0, current version 24.1.0)
```
and matlab 2019b have freetype 23
```
@rpath/libfreetype.6.dylib (compatibility version 23.0.0, current version 23.0.0)
```

So here is my question, Is it the compatibility issue (different version of fontconfig and freetype), or the absolute path, caused the 'Library not loaded' error? Is it possible to make OFTGL work without installing XQuartz?

**Comment by kleinerm** [comment]
See this commit from 2 months ago, which should fix it for the upcoming beta:
https://github.com/kleinerm/Psychtoolbox-3/commit/6b9bb2cad4b0213885c075dbb116c00e0e7de51e

**Comment by kleinerm** [comment]
Should be now fixed in new PTB 3.0.17.
